README.md

```diff
@@ -258,41 +275,6 @@ nix_direnv_watch_file mod1.nix mod2.nix
 use flake
 ```
 
-## Shell integration
-
```

Comment:
## Shell integration

See the [wiki](https://github.com/nix-community/nix-direnv/wiki/Shell-integration) for helpers to quickly setup a direnv setup in a new project.